### DISCUSSION TITLE
Models being used?

### DISCUSSION THREAD

**Discussion by pstadnyk** [TencentARC/PhotoMaker#95] [General]
I am particularly interested in the model that yields "Comic Book" style. Can someone tell me where this style is coming from so I can load it also into my A1111?

**Comment by Paper99** [comment]
Hi, @pstadnyk. Please see this prompt setting https://github.com/TencentARC/PhotoMaker/blob/main/gradio_demo/style_template.py#L42.
It works for many base models based on SDXL.

**Reply by pstadnyk** [reply]
I've tried but I just can't seem to get the same style results simply with A1111 Img2Img + your prompt + an XL model. Is there something I'm missing?

**Reply by Paper99** [reply]
Maybe you need to try a different base model, which specializes in stylization.
